es-es/erlang-es.html.markdown

```diff
@@ -0,0 +1,292 @@
+---
+language: Erlang
+lang: es-es
+contributors:
+    ["Giovanni Cappellotto", "http://www.focustheweb.com/"]
```

Comment:
Missing `-` here before contributor name,, fix as follows -
```
contributors:
    - ["Giovanni Cappellotto", "http://www.focustheweb.com/"]
```

Reply:
thank you, I will work on it